### DISCUSSION TITLE
Chat with multiple attachments

### DISCUSSION THREAD

**Discussion by SR--** [lifan0127/ai-research-assistant#67] [General]
Thanks for the tool, just trying it...

How can one chat with multiple attachments? E.g., with a Zotero collection? Or even the whole library? Is it possible?

Also, is it only PDFs? What about epubs?

**Comment by MohamedNumair** [comment]
I can't make it access my PDF's, I am not using Zotero's main storage, I am using an external server, but the pdfs are available locally on my computer, however, ARIA seems to not read them at all. or am I missing something?

**Reply by lifan0127** [reply]
Hi @MohamedNumair currently ARIA does not read PDF attachments. This feature is planned for the next major release. Thanks!
